Failing to compile

On the Install dependencies check, it is saying:
```
Run bundle install
  bundle install
  shell: /bin/bash -e {0}
/opt/hostedtoolcache/Ruby/2.7.0/x64/lib/ruby/2.7.0/rubygems.rb:275:in `find_spec_for_exe': Could not find 'bundler' (1.17.3) required by your /home/runner/work/circuitpython-org/circuitpython-org/Gemfile.lock. (Gem::GemNotFoundException)
To update to the latest version installed on your system, run `bundle update --bundler`.
To install the missing version, run `gem install bundler:1.17.3`
	from /opt/hostedtoolcache/Ruby/2.7.0/x64/lib/ruby/2.7.0/rubygems.rb:294:in `activate_bin_path'
	from /opt/hostedtoolcache/Ruby/2.7.0/x64/bin/bundle:23:in `<main>'
##[error]Process completed with exit code 1.
```

PR coming soon

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -34,6 +34,7 @@ jobs:
         ruby-version: '2.x'
     - name: Install Dependencies
       run: |
+        gem install bundler:1.17.3
         bundle install
     - name: Build site with jekyll
       run: |

